### PR TITLE
Updates to ISLE behat suite to facilitate institutional tests

### PR DESCRIPTION
**Pull request by noahwsmith**
We have made the following changes to the universal behat suite to allow it to be run on a higher percentage of customized themes. We've removed assumptions which are specific to the plain vanilla Islandora 7 theme.

Changes:

- Removed portainer test since many folks won't run that in all environments
- Added an `@disabled` tag to allow folks to easily disable unwanted features
- Added tags to run for cleanup, to recover from failed test runs `@teardown-only` and `@teardown-full`
- Removed theme-specific search results phrasing tests
- Made the "find" command work better on customized themes

### CODE REVIEW

**Comment by noahwsmith** [comment]
This behat run should prove that these tests do not introduce any failures on generic sites: https://app.circleci.com/jobs/github/Born-Digital-US/ISLE/349

**Comment by noahwsmith** [comment]
New CI run to prove it still works on generic sites: https://app.circleci.com/jobs/github/Born-Digital-US/ISLE/351

**Comment by noahwsmith** [comment]
Whoops, needed another delay for Solr indexing. Rerunning tests: https://app.circleci.com/jobs/github/Born-Digital-US/ISLE/352

**Comment by noahwsmith** [comment]
Fedora crashed in Circle - running again with no changes: https://app.circleci.com/jobs/github/Born-Digital-US/ISLE/353

**Comment by noahwsmith** [comment]
That last run passed (#353) - I recommend accepting this PR based on that, @dwk2

**Review by dwk2 — APPROVED** [review]
This PR looks good to me. Thank you @noahwsmith !

File: features/compound.feature [hunk 1]
```diff
@@ -86,7 +84,7 @@ Feature: Test Compound Object CModel
     And I wait for AJAX to finish
     Then I press "Next"
     And wait for the page to be loaded
-    Then I should see "MARCXML File"
+    # Then I should see "MARCXML File"
```
Comment (bseeger) [inline features/compound.feature]: do you just want to take these lines out, or is it valuable to have them there as a comment?

Comment (noahwsmith) [reply features/compound.feature]: removed - thanks!

**Comment by bseeger** [comment]
Looks good to me too.  Thanks @noahwsmith!

**Pull request merged**